Game/GameInfo.pde

```diff
@@ -74,6 +78,10 @@ public class GameInfo {
   private int playerScore1;
   private int playerScore2;
   private boolean networkSessionClosing;
+  
+  private boolean ghostMagnetActive = false;
+  private float magnetX = 0.0;
+  private float magnetY = 0.0;
```

Comment:
Better not to place these three variables inside the `GameInfo` class. The `GameInfo` class should not participate in the game logic.

I would suggest create another SynchronizedItem for the "magnet".
Or, if want a more convenient way, place it inside the `PagePlay` class.

Reply:
> I would suggest create another SynchronizedItem for the "magnet".
> Or, if want a more convenient way, place it inside the `PagePlay` class.

Currently I have no idea how Ghosts get the state of Magnet under these two ways.

Reply:
Don't worry. You can implement the magnet as such for now. Later I can modify the code.

Reply:
Seems like I have successfully modified, I will submit a new version, you can check it again.